src/app/components/profile/modal/skills/skills.scss

```diff
@@ -71,3 +73,4 @@
         }
     }
 }
+}
```

Comment:
nope, wrong indentation somewhere then, cant have two closing ones at the end with the same indentation.